public/utils/constants.js

```diff
@@ -62,7 +62,7 @@ export const MONITOR_INPUT_DETECTOR_ID = `inputs.${INPUTS_DETECTOR_ID}`;
 
 export const AD_PREVIEW_DAYS = 7;
 
-export const MAX_QUERY_RESULT_SIZE = 200;
+export const MAX_QUERY_RESULT_SIZE = 5000;
```

Comment:
Changing this will update the max size for all the api calls, do we only want to change it for the channels?

Reply:
Right, I added MAX_CHANNELS_RESULT_SIZE=5000 to be used just for getting all the channels.